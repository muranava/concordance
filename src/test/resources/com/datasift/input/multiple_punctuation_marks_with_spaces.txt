Would you like to go for a coffee? ! ? I would.. . But I can't.