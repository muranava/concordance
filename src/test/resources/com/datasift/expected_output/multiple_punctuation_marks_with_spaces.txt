a {1:0}
but {1:2}
can't {1:2}
coffee {1:0}
for {1:0}
go {1:0}
i {2:1,2}
like {1:0}
to {1:0}
would {2:0,1}
you {1:0}